ui/app/abilities/job.js

```diff
@@ -19,6 +19,14 @@ export default class Job extends AbstractAbility {
   )
   canScale;
 
+  @or(
+    'bypassAuthorization',
+    'selfTokenIsManagement',
+    'specificNamespaceSupportsReading',
+    'policiesSupportReading'
+  )
+  canRead;
```

Comment:
This seems out of place in a PR like this, right?

When we save a variable, there are a few things that we want to update and depend on that change. One of those is `job.pathLinkedVariable`, which looks against a job's `variables` array (more on that later!) and finds the one that's named `nomad/jobs/$jobname`.

So now, in our saveVariable() function elsewhere, we can say "Oh great you saved! Do you have access to read jobs so that this might matter for them? Yes? Okay, update them."

This provides the ability to check that they can read jobs.